src/lib/InstantSearch.ts

```diff
@@ -277,6 +277,16 @@ See ${createDocumentationLink({
     return this;
   }
 
+  // @major we shipped with EXPERIMENTAL_use, but have changed that to just `use` now
+  public EXPERIMENTAL_use(...middleware: Middleware[]): this {
+    warning(
+      false,
+      'We recommend you to replace `EXPERIMENTAL_use` with `use`.'
```

Comment:
Let's explain _why_ to users.

```suggestion
      'The InstantSearch middleware API is now considered stable so we recommend replacing `EXPERIMENTAL_use` with `use` before upgrading to the next major version.'
```